cmd/pulumi-test-language/l1empty_test.go

```diff
@@ -291,9 +294,8 @@ func TestL1Empty_BadSnapshot(t *testing.T) {
 	t.Logf("stdout: %s", runResponse.Stdout)
 	t.Logf("stderr: %s", runResponse.Stderr)
 	assert.False(t, runResponse.Success)
-	assert.Equal(t, []string{
-		"program snapshot validation failed:\nexpected file Pulumi.yaml does not match actual file",
-	}, runResponse.Messages)
+	assert.Equal(t, 1, len(runResponse.Messages))
+	assert.Contains(t, runResponse.Messages[0], "program snapshot validation failed")
```

Comment:
This can be an equals can't it? 
```suggestion
	assert.Equal(t, program snapshot validation failed\nexpected file Pulumi.yaml does not match actual file",  runResponse.Messages[0])
```

Reply:
Ah no, it should be contains cause we get the diff now. I'll fix this in my PR

Reply:
I think that's why I made it `Contains`